src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/wwwroot/styles/styles.scss

```diff
@@ -45,7 +45,7 @@ p {
     }
 }
 
-.bit-lnk-fluent > a {
+.navbar > .bit-lnk-fluent > a {
     float: left;
```

Comment:
why float? I'm not comfortable with it.
and there is another float attribute in this selector, please investigate and improve it.

Reply:
I meant the "float" attribute, not the selector's name.